packages/client/src/pages/admin/emoji-edit-dialog.vue

```diff
@@ -27,85 +27,75 @@
 </XModalWindow>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script lang="ts" setup>
+import { } from 'vue';
 import XModalWindow from '@/components/ui/modal-window.vue';
 import MkButton from '@/components/ui/button.vue';
 import MkInput from '@/components/form/input.vue';
 import * as os from '@/os';
 import { unique } from '@/scripts/array';
+import { i18n } from '@/i18n';
 
-export default defineComponent({
-	components: {
-		XModalWindow,
-		MkButton,
-		MkInput,
-	},
+const props = defineProps({
+	emoji: {
+		required: true,
+	}
+});
 
-	props: {
-		emoji: {
-			required: true,
-		}
-	},
+let dialog = $ref(null);
+let name: string = $ref(props.emoji.name);
+let category: string = $ref(props.emoji.category);
+let aliases: string = $ref(props.emoji.aliases?.join(' '));
```

Comment:
According to the [type definition in the Misskey SDK](https://github.com/misskey-dev/misskey.js/blob/c17c42fa05bcbcc38c7a4395d1bb3d2336d3d402/src/entities.ts#L252), `aliases` can not be null.
```suggestion
let aliases: string = $ref(props.emoji.aliases.join(' '));
```